crates/bitwarden/src/platform/fido2/mod.rs

```diff
@@ -1,24 +1,37 @@
-use crate::{
-    error::Result,
-    vault::{login::Fido2Credential, CipherView},
-    Client,
-};
+use std::sync::Mutex;
+
+use bitwarden_crypto::{KeyContainer, SensitiveString, SensitiveVec};
+use passkey::types::{ctap2::Aaguid, Passkey};
 
 mod authenticator;
 mod client;
+mod crypto;
 mod traits;
+mod types;
 
-pub use authenticator::{
-    Fido2Authenticator, GetAssertionRequest, GetAssertionResult, MakeCredentialRequest,
-    MakeCredentialResult,
+pub use authenticator::Fido2Authenticator;
+pub use client::Fido2Client;
+pub use traits::{
+    CheckUserOptions, CheckUserResult, Fido2CredentialStore, Fido2UserInterface, Verification,
 };
-pub use client::{
-    AuthenticatorAssertionResponse, AuthenticatorAttestationResponse, ClientData, Fido2Client,
+pub use types::{
+    AuthenticatorAssertionResponse, AuthenticatorAttestationResponse, ClientData,
+    GetAssertionRequest, GetAssertionResult, MakeCredentialRequest, MakeCredentialResult,
     PublicKeyCredentialAuthenticatorAssertionResponse,
     PublicKeyCredentialAuthenticatorAttestationResponse,
 };
-use passkey::types::Passkey;
-pub use traits::{CheckUserOptions, CheckUserResult, CredentialStore, UserInterface, Verification};
+
+use self::crypto::{cose_key_to_pkcs8, pkcs8_to_cose_key};
+use crate::{
+    error::{Error, Result},
+    vault::{CipherView, Fido2CredentialFullView, Fido2CredentialView},
+    Client,
+};
+
+// AAGUID: d548826e-79b4-db40-a3d8-11116f7e8349
```

Comment:
We should mention what the AAGUID is for.

Reply:
Done in 55938d63d952e989096f8d00062b23b781695b48